assets/script/index.js

```diff
@@ -0,0 +1,28 @@
+// Check point for inputed string
+function palindromeText (str) {
+  let j = str.length - 1
+  for (let i = 0; i < j / 2; i++) {
+    if (str[i] !== str[j]) {
+      return false
+    }
+    j--
+  }
+  return true
+}
+
+function checkPalindrome () {
```

Comment:
This function doesn't work for the phrase `do geese see god` which is a palindrome, could you check the logic again?

Reply:
finally got it